templates/profile.html

```diff
@@ -0,0 +1,66 @@
+<html>
+<head>
+  <title>PicForEating</title>
+
+  <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.4.min.js"></script> 
+  <script type="text/javascript" src="public/application.js"></script>
+  <script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+  
+  <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <link href="public/custom2.css" rel="stylesheet">
+  <link rel="icon" href="public/images/favicon.ico">
+  <link href='https://fonts.googleapis.com/css?family=Give+You+Glory' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Alegreya+Sans+SC:300' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Alegreya+Sans+SC:300italic' rel='stylesheet' type='text/css'>
+</head>
```

Comment:
you should try and implement the layouts so you don't have to repeat over and over again the html boilerplate